README.md

```diff
@@ -1 +1,176 @@
-# metrics-webapp
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Country weather app <a name="about-project"></a>
+
+<br>
+
+<br>
+"Country weather app" is a webapp for everyone who is interested in weather and would like to check all the metrics including temperature, wind speed, humidity, etc. It is a Single Page App (SPA) that allows users to:<br>
+- Choose a country to dive deeper into its weather metrics.<br>
+- When clicked on a country it displayes all the metrics of that country capital's in regards of weather.
+
+<br>
+<br>
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML.
+- CSS.
+- JavaScript
+- LINTERS.
+- REACT.
+
+<br>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+<br>
+
+- **Project is created using React**
+- **Data has been fetched and displayed on the app from http://api.weatherstack.com/ with different endpoints**
+- **Easy to reuse components for future projects**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<br>
+
```

Comment:
@Gambit142 i would like to ask you regarding deploying my page. i have used render to deploy it, and i do recieve an error 403 in the console and page displays failed to fetch data. error 403 is Authentication Failure, which i believe is due to API i have chosen. the chosen API was from the list given in github repo under the section weather. could you advise me what i should do in this situation? Do you think it would work without issuess if i would deploy the project using guthub pages?

Reply:
hi @Gambit142 i have tried multiple ways to deploy the project, unfortunately the problem is the API itself, as i have recieved a message that i have to subscribe to a different plan (paid for) to be able to deploy the project that shows fetched data as shown in the image

![Screenshot_2023-09-06_18-52-17](https://github.com/agneta1991/metrics-webapp/assets/120168752/2576f8ed-af4b-4234-bfd5-129fe2630492)

i also have tried implementing a meta tag in the index.html which should resolve this issue, but i cant faind the way to work around it.

Reply:
@Gambit142 could you please get back to me as soon as you can. As I am slightly lost what I should do in this situation

Reply:
Hi @agneta1991, Since it is a requirement, I would advise that you look for an API that doesn't require you to pay before you can deploy your application. Most of the changes suggested are based on design and technical requirements, so you can just plug in your API to fit the current design but you might just need to change your theme.

Reply:
@Gambit142 the API was provided in GitHub repo, it's not like I have chosen something outside what has been suggested inside requirements. I have implemented all the changes required and tests already, how I can change the API now when the while project is complete apart from deploying it? That's not right. I'm in touch with senior developer as well as Zendesk. You can't require me to re-do whole project one day before the deadline over something that has nothing to do with the functionality of the code. Requirement says deploy it, which I have done, it's outside of my control what it display..

Reply:
Hi @agneta1991, I am not requiring you. Please. It's just my suggestion and advice. Please note this. The only requirements are based on the changes I suggested. I am sorry if you think my [comment](https://github.com/agneta1991/metrics-webapp/pull/1#discussion_r1318124884) is a requirement. It was just an advice.